### PR TITLE
Use generic Github API to download

### PR DESCRIPTION
**Pull request by TommyE123**

### CODE REVIEW

File: jackett/jackett-update.sh [hunk 1]
```diff
@@ -13,7 +13,8 @@ if DoesAppFolderExist; then
     source "$SCRIPTPATH/inc/app-install-deps.sh"
     source "$SCRIPTPATH/inc/app-folders-create.sh"
     if "$SCRIPTPATH/jackett/jackett-version-handler.sh"; then
```
Comment (nemchik) [inline jackett/jackett-update.sh]: Shouldn't this now be `inc/app-git-latest-release-version.sh`?

Comment (nemchik) [reply jackett/jackett-update.sh]: Nevermind, I see what's going on. Deleting comments

**Comment by nemchik** [comment]
So,just looking at these, aren't all the app specific version check files able to be made into a single global version handler?

**Comment by TommyE123** [comment]
Yes I thought that as well. But doesn't Ombi need to do more work then the others? I can look at consolidating them though for sure. I think this is a first good step though:-)

**Comment by nemchik** [comment]
I don't think Ombi needs to do more. Sonarr/Radarr/Jackett don't actually pull the current version from the app do they? We're just writing the installed version to `ServiceVersions.cfg` which is determined by the the version downloaded using `inc/app-git-latest-release-version.sh` I thought? In Ombi's case that would work the same way.

**Pull request merged**